prow/knative/config.yaml

```diff
@@ -173,6 +174,18 @@ tide:
     missingLabels: *knative_tide_missing_labels
     author: knative-prow-robot
     reviewApprovedRequired: false
+  - repos:
+    - knative-sandbox/net-contour
+    labels:
+    - lgtm
+    - approved
```

Comment:
will github reviews make the approved label? I don't think so, but then will prow add approved label from reviews? I thought that is what @grantr added.

Reply:
I think prow will add approved labels, but your change seems to make this repo requires `lgtm` and `approve` label, as well as explicit github code review approval, is this desired?

Reply:
yes, that is the goal